packages/menu-bar/test/menu-bar.test.js

```diff
@@ -456,6 +456,71 @@ describe('overflow button', () => {
   });
 });
 
+describe('has-single-button attribute', () => {
+  let menu, buttons;
+
+  beforeEach(async () => {
+    menu = fixtureSync('<vaadin-menu-bar></vaadin-menu-bar>');
+    menu.items = [{ text: 'Item 1' }, { text: 'Item 2' }, { text: 'Item 3' }, { text: 'Item 4' }];
+    await nextRender(menu);
+    buttons = menu._buttons;
+  });
+
+  it('should not set when at least one button is visible', async () => {
```

Comment:
I would clarify somehow that this case is about **the overflow button** + one visible button (as opposed to only one visible button), or add an assertion that the overflow button is also visible.

```
should not set when one button and overflow are only visible
```